source/draw/RasterImage.ooc

```diff
@@ -66,17 +66,32 @@ RasterImage: abstract class extends Image {
 		} else {
 			originalPen := pen
```

Comment:
Is this line some residual of how `Pen` used to work? Looks like we can remove it.